__tests__/server/devHolocronCdn.spec.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
curious why it went from `devHolocronCDN.spec.js` to `devHolocronCdn.spec.js`, I believe that that file name change makes github treat them as different files (one being deleted and another as new)

Reply:
Yeaaa see what happened is that I deleted it originally I believe thinking that it was not needed for coverage but then I re added it not undoing the deletion but re adding the file and I guess I messed up the name

Reply:
it would be great to maintain the same name since the tested file is actually called `devHolocronCDN` and to get a diff from github and see what have changed (easier to review)